docker/Dockerfile-workers

```diff
@@ -34,10 +34,14 @@ RUN dnf -y install \
     python3-devel \
     python3-pip \
     skopeo \
+    net-tools \
```

Comment:
Do not forget to modify Ansible role iib-role in our gitlab instance or you will miss this package on stage/prod. 
This Dockerfile is used just for local development If I am not mistaken.

Reply:
I have created additional MRs